app/views/content/funding-and-support/if-youre-disabled.md

```diff
@@ -45,29 +55,38 @@ $international-content$
 
 ### Disabled Students' Allowance (DSA)
 
-You can [apply for Disabled Students’ Allowance (DSA)](https://www.gov.uk/disabled-students-allowance-dsa/how-to-claim) to cover the study-related costs you have because of a mental health problem, long term illness or any other disability.
+If you’re on a fee-paying teacher training course, you may be eligible for Disabled Students’ Allowance (DSA). You cannot get DSA if your course pays a salary. 
 
-You’ll get confirmation of whether your application has been successful within 6 weeks.
+DSA can help towards covering any study-related costs you have because of a mental health problem, long term illness or any other disability. You can get DSA on its own or in addition to other student finance. 
```

Comment:
```suggestion
DSA can help cover any study-related costs you have because of a mental health problem, long-term illness or any other disability. You can get DSA on its own or in addition to other student finance. 
```